site/components/pages/Infinity/styles.ts

```diff
@@ -833,10 +824,60 @@ export const ctaSection = css`
   }
 `;
 
-export const footerWrapper = css`
-  grid-column: full;
-  grid-template-columns: inherit;
-  footer {
-    background-color: black;
+export const footerContainer = css`
+  padding-bottom: 3rem;
+  padding-top: 5rem;
+  grid-template-rows: 1fr 50px 1fr;
```

Comment:
we avoid px, always rem, so that it scales if user changes their browser default font size

so 4.8rem or 5.6rem here should work

Reply:
Sure thing, I've updated.